ymazniou/JQuery/JQtabl.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Jquery Sample</title>
+	<style>
+		.zebra {
+			background-color: grey;
+			color: white;
+		}
+    </style>
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+	<script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
+	<script>
+	$(document).ready(function(){
+        $('#table_data').css('border-collapse','collapse').css('margin-left','20px');
+        $('#table_data tr:even').css('background-color','silver');
+        $('tr:first-child').css('background-color','gray').css('color','white');
+        $('tr:first-child th:first-child').css('text-align','center');
+        $('td').css('border','dotted black 1px');
+        $('td:last-child').css('border-right','none');
+        $('th').css('border-bottom','1px dotted black');
+        $('caption').css('text-align','center').css('font-variant','small-caps').css('margin-top',"0px").css('padding-top','0px');
```

Comment:
What is the train here?
All classes you can get form boostrap
Like the following example:
$('#table_data').addClass('table table-hover table-light');
$('.display_button').addClass('btn btn-primary btn-sm  btn-block disabled');